app/mailers/user_notifications.rb

```diff
@@ -444,6 +444,16 @@ def send_notification_email(opts)
         else
           I18n.t('subject_pm')
         end
+
+      participants = "#{I18n.t("user_notifications.pm_participants")} "
+      participant_list = []
+      post.topic.allowed_users.each do |user|
+        participant_list.push user.username
```

Comment:
I am not sure this is correct, if `prioritize username in ux` should we not be showing "name" here, also should we make this hyperlinks to the actual user?